portal-web/docroot/html/taglib/ui/input_field/page.jsp

```diff
@@ -513,7 +513,15 @@ if (hints != null) {
 							/>
 						</c:when>
 						<c:otherwise>
-							<input <%= Validator.isNotNull(autoComplete) ? "autocomplete=\"" + autoComplete + "\"" : StringPool.BLANK %> class="<%= cssClass %> lfr-input-text" <%= disabled ? "disabled=\"disabled\"" : StringPool.BLANK %> id="<%= namespace %><%= id %>" name="<%= namespace %><%= fieldParam %>" <%= Validator.isNotNull(placeholder) ? "placeholder=\"" + LanguageUtil.get(resourceBundle, placeholder) + "\"" : StringPool.BLANK %> style="<%= upperCase ? "text-transform: uppercase;" : StringPool.BLANK %>" maxLength="<%= maxLength %>" type="<%= secret ? "password" : "text" %>" value="<%= autoEscape ? HtmlUtil.escape(value) : value %>" />
+							<c:if test="<%= upperCase %>">
+								<aui:style type="text/css">
+									#<%= namespace %><%= id %> {
+										text-transform: uppercase;
+									}
```

Comment:
Normally we don't put IDs in CSS, as IDs are often autogenerated, sometimes even set by external JS. But here, as it follows JSP logic, it should be fine :+1: